<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tools::FspPrinterDialogPatient</class>
 <widget class="QWidget" name="Tools::FspPrinterDialogPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>278</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>774</width>
    <height>278</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>774</width>
    <height>278</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>278</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>774</width>
     <height>278</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>774</width>
     <height>278</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>774</width>
     <height>278</height>
    </size>
   </property>
   <property name="pixmap">
    <pixmap>../../../global_resources/pixmap/others/S3110_patient.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="assureName">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>190</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>88</y>
     <width>571</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="assureNss">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>223</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
   <property name="inputMask">
    <string notr="true"># ## ## ## ### ### ##; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="address">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>260</y>
     <width>571</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
  </widget>
  <widget class="Utils::ModernDateEditor" name="dob">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>151</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="orga">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>120</y>
     <width>179</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
   <property name="inputMask">
    <string>N N N N N N N N N; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nns">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>120</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
   <property name="inputMask">
    <string notr="true"># ## ## ## ### ### ##; </string>
   </property>
  </widget>
  <widget class="Utils::ModernDateEditor" name="billDate">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>33</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="billId">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; background: transparent;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::ModernDateEditor</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/widgets/moderndateeditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>billId</tabstop>
  <tabstop>billDate</tabstop>
  <tabstop>name</tabstop>
  <tabstop>nns</tabstop>
  <tabstop>dob</tabstop>
  <tabstop>assureName</tabstop>
  <tabstop>assureNss</tabstop>
  <tabstop>address</tabstop>
  <tabstop>orga</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
